Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py

```diff
@@ -424,11 +478,32 @@ def build_where_clause(args: dict) -> str:
         'file_sha_256': 'file_sha_256',
         'file_name': 'file_name',
     }
+    if args.get('ip') and (args.get('source_ip') or args.get('dest_ip')):
+        raise DemistoException('Error: "ip" argument cannot appear with either "source_ip" nor "dest_ip"')
+
+    if args.get('port') and args.get('source_port') or args.get('port') and args.get('dest_port'):
```

Comment:
use the same condition logic as in line 481

Reply:
done